### ISSUE TITLE
audit-avt1-breadcrumb: h3 id in use - color, typography, id

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-website#1591]
## Environment:

> Operating system: Mac
> Browser: Chrome 
> AVT :one:
> Page: [Components - Breadcrumb: style](https://www.carbondesignsystem.com/components/breadcrumb/style/)
> This issue affects 3 total violations

## Detailed Description: 

 h3 id in use - color, typography, id

<img width="1680" alt="Screen Shot 2020-08-20 at 11 12 50 AM" src="https://user-images.githubusercontent.com/32720851/90790793-727e9b00-e2d6-11ea-8268-54b377374f52.png">

## What to do: 
For each element with an id attribute, provide a unique, non-empty value.

## Why is this important?
The id attribute provides a machine-readable identifier for the elements of the page. A unique value for id allows browsers and assistive technologies to interpret and parse the content accurately.

**Comment by andreancardona** [comment]
Fix can be seen here: https://github.com/carbon-design-system/carbon-website/pull/2150
